### PR TITLE
Added French Translation strings

### PR DESCRIPTION
**Pull request by GautamMandsorwale**
Hi @maik-kulbe,

Would request you to please verify and approve this pull request.
I have added French Translation strings

- Gautam Mandsorwale

### CODE REVIEW

**Comment by GautamMandsorwale** [comment]
@maik-kulbe 

Can you please guide me as to what went wrong on below CI review?

**Review comment :** _continuous-integration/travis-ci/pr — The Travis CI build failed_

File: lang/fr.php [hunk 1]
```diff
@@ -0,0 +1,143 @@
+<?php
+
+$lang['title'] = 'TecArt Ticket System';
+
+$lang['foot_note'] = 'Ticket-Web Interface &copy; by TecArt GmbH';
+
+$lang['custom_number']  = 'N ° de client';
```
Comment (ghost) [inline lang/fr.php:7]: This looks a little odd with the space and the degree(?) symbol. I'm not as good with french i10n but wouldn't it be better to just write 'numéro' as full word?

**Comment by ghost** [comment]
@GautamMandsorwale don't worry, the Travis integration is currently returning errors for master and for your file it reports just a few info level notifications that are nothing serious: https://travis-ci.org/TecArt/TecArt-Ticket-System/jobs/292647801#L3401-L3407 

I've added a comment to your translations, apart from that I can't see a reason why things shouldn't be merged

File: lang/fr.php [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+$lang['password']       = 'Mot de passe';
+$lang['login']          = 'S\'identifier';
+$lang['headline_standard'] = 'Ticket-Web Interface';
+$lang['headline']       = 'TecArt Ticket-Web Interface';
+$lang['logout']         = 'Connectez - Out';
+$lang['view']           = 'Vue';
+
+$lang['show_current']       = 'Billets actuels';
```
Comment (ghost) [inline lang/fr.php:15]: Same as in #11 - is "billets" the correct word for incident tickets?

File: lang/fr.php (hunk 1 above)
Comment (sea75300) [reply lang/fr.php:7]: It's common french, but should be "N° de client".

File: lang/fr.php (hunk 2 above)
Comment (sea75300) [reply lang/fr.php:15]: "billets" is common french too

**Review by sea75300 — CHANGES_REQUESTED** [review]
Change "N°" to "N°"

**Review by sea75300 — APPROVED** [review]

**Pull request merged**